serving/docker/partition/sm_neo_shard.py

```diff
@@ -103,6 +103,7 @@ def shard_lmi_dist_model(self, input_dir: str, output_dir: str,
             enforce_eager=True,
             disable_custom_all_reduce=True,
             distributed_executor_backend="mp",
+            gpu_memory_utilization=0.99,
```

Comment:
While this may allow for the sharding of 405b on a p4de, will the resulting converted artifacts even be usable by the customer on the same instance? Do we also provide the necessary configs in the output to ensure the model can be loaded on the instance for inference?

Reply:
That is correct, it would have to be set to the same value during runtime. I updated the PR to simply allow customers to specify `option.gpu_memory_utilization` during sharding jobs, and with #2545 that value will be propagated to the `serving.properties`. Same with `option.enforce_eager`, since it also affects GPU mem utilization.


Thus customers wanting to use a model/instance combination which requires a change to `gpu_memory_utilization` or `enforce_eager` will have to be aware of this at the time of converting the artifacts.

Reply:
Awesome, this sounds like the right approach!